docs/embedding/static-embedding-parameters.md

```diff
@@ -165,6 +165,17 @@ You can preview appearance settings from your question or dashboard's [embedded
 
 \* Available on [Pro](https://www.metabase.com/product/pro) and [Enterprise](https://www.metabase.com/product/enterprise) plans.
 
+### Download results
+
+{% include plans-blockquote.html feature="Downloading results" %}
+
+You can enable or disable downloading results for embedded questions by setting the `hide_download_button` parameter in the embedding URL to `false` or `true`.
+
+If the download button is missing when you expected it to be available, check that the URL in the `src` attribute for your iframe has the parameter `hide_download_button=false`.
```

Comment:
Is this line necessary? Won't it also work if that parameter isn't in the URL?

Reply:
This was the problem that several customers had: they said the download button was missing, and when we asked them to check the URL, turned out they had this parameter set to true.  I'm open to rephrasing, but i would like to keep the general message of "check the URL if the embedding not behaving as you expect" explicitly written